modulemd/v2/modulemd-yaml-util.c

```diff
@@ -392,6 +392,35 @@ modulemd_yaml_parse_string (yaml_parser_t *parser, GError **error)
 }
 
 
+gboolean
+modulemd_yaml_parse_bool (yaml_parser_t *parser, GError **error)
+{
+  MMD_INIT_YAML_EVENT (event);
+
+  YAML_PARSER_PARSE_WITH_EXIT_BOOL (parser, &event, error);
+  if (event.type != YAML_SCALAR_EVENT)
+    {
+      MMD_YAML_ERROR_EVENT_EXIT_INT (
+        error, event, "Expected a scalar boolean");
+    }
+
+  if (g_strcmp0 ((const gchar *)event.data.scalar.value, "false"))
```

Comment:
Oh, isn't there some better way to handle booleans?

Reply:
Could you be more specific? We really only get them as a scalar value. I dabbled with the idea of doing a case-insensitive compare, but that seemed like overkill.

We can always enhance this function later, that's why it's separate.

Reply:
FALSE is not a boolean by yaml specification

I was thinking whether there is libyaml function to get bool

Reply:
Nothing public: https://github.com/yaml/libyaml/blob/master/include/yaml.h